src/utilities/util_dump.c

```diff
@@ -155,7 +155,7 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
          * case, we're specifically interested in what is visible at a given read timestamp.
          */
         if (WT_STREQ(simpleuri, WT_HS_URI) && timestamp == NULL)
-            F_SET(session_impl, WT_SESSION_IGNORE_HS_TOMBSTONE);
+            F_SET(cursor, WT_CURSTD_IGNORE_TOMBSTONE);
```

Comment:
It wasn't you that added this - but the utility implementations are generally considered external applications and the WT_SESSION_IMPL structure and the associated flags (like that here) are internal.
Could you open a ticket to "Review the utility code and make sure it works as an external application"?

Reply:
Created WT-6112